content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
Consider renaming the file simply `cloud-foundry.md` (unless you're planning to submit another blog post in 2023 :)).